src/Plugin.php

```diff
@@ -43,7 +43,7 @@ public static function init() {
     // Ensure that Facebook embeds do not exceed available content width.
     add_filter('oembed_fetch_url', __CLASS__ . '::oembed_fetch_url', 10, 3);
 
-    // Add wrapper to oEmbed elements.
+    // Enforce youtube-nocookie and add wrapper to oEmbed elements.
```

Comment:
In the other case yes, but not here. The verb in third person form only applies to PHPDoc summary lines, not to inline comments.

A function does something for you (because you invoke it).

Regular code in a function body is doing something for some reason that is being documented in inline comments.

Reply:
Thanks @sun! I will try to remember that.